### ISSUE TITLE
Releases no longer available for AppVeyor deployement

### ISSUE THREAD

**Issue by chrisofspades** [Wyamio/Wyam#9]
I'm using AppVeyor to deploy my site as described here:

https://wyam.io/docs/deployment/appveyor

And it's failing on the step to download the latest release. See log here:

https://ci.appveyor.com/project/chrisofspades/chrispeoples-com/builds/31172904

Looks like all releases/tags were removed from the repo. Any chance of bringing those back?

**Comment by daveaglick** [comment]
> Looks like all releases/tags were removed from the repo. Any chance of bringing those back?

Yes, absolutely. It's on the top of my to-do list - hoping to get some time tonight to work on it. The releases was an unexpected goof-up on my part when I moved the original project to https://github.com/statiqdev/Statiq.Web. I transferred relevant issues back here but totally forgot about the releases. Shouldn't be too hard to write a little script to open them back up here where they should be and delete them over there.
